src/Table/Table.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { StyledTable } from "./table.styles";
+
+const tableWithDefaults = props => {
+  const defaultOptions = {
+    showPagination: false,
+    minRows: 0,
+    sortable: false,
+    resizable: false
+  };
+
+  return <StyledTable {...defaultOptions} {...props} />;
+};
+tableWithDefaults.propTypes = StyledTable.propTypes;
```

Comment:
`StyledTable.propTypes;` having hard time understand what is this value. Can you please explain :)

Reply:
Here we just use the existing `ReactTable` proptypes. As we are just wrapping `ReactTable` and setting some sensible defaults, we allow all proptypes from `ReactTable`. If we extend this later to blacklist or add some more propTypes we can do stuff like this:
```
tableWithDefaults.propTypes = {
    ...StyledTable.propTypes, 
    extraProp: PropTypes.string.isRequired
};
```